com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/Deferred.compute

```diff
@@ -34,6 +34,8 @@
 
 #pragma multi_compile _ SHADOWS_SHADOWMASK
 
+#pragma multi_compile SHADOW_LOW SHADOW_MEDIUM SHADOW_HIGH
```

Comment:
you need to do this also for deferred.shader and tiledeferred.shader

Reply:
Ha! Yes I always forget about those two.  Will do!